includes/wikia/services/UserService.class.php

```diff
@@ -62,6 +66,8 @@ public function getUsers( $ids ) {
 	 * means the method User::isPasswordReminderThrottled will return true for the next
 	 * $wgPasswordReminderResendTime hours
 	 *
+	 * @todo remove after we deprecate temporary passwords
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)